### PR TITLE
Feat/iml

### PR DESCRIPTION
**Pull request by IVANDOMENZAIN**

### CODE REVIEW

**Comment by IVANDOMENZAIN** [comment]
@mihai-sysbio I've seen that you have merged the base branch in the fork repo to the SysBio repo. Last week I was able to get the pipeline running for `iML1515` returning the expected results. My work there started from commit 68374b6 on. Additionally, I've solved some conflicts in the `workflow` and `config` files in order to enable the merge of this PR. I suggest that all this is integrated into `SysBioChalmers: feat/update_all_ecModels`.

**Comment by IVANDOMENZAIN** [comment]
@mihai-sysbio Additionally, in this PR the download of a model in `.xml` format from an external URL, and its successful loading as a MATLAB structure (with RAVEN) were enabled.

File: tools.py [hunk 1]
```diff
@@ -57,8 +57,8 @@ def cleanup(self, section, subdir=''):
     def scripts(self, gem):
         return '{}/{}'.format(gem, SCRIPTSDIR)
 
-    def mat_file_location(self, gem):
-        return self.install_dir(gem) + self.config[gem]['mat_filename']
+    def model_file_location(self, gem):
```
Comment (mihai-sysbio) [inline tools.py:60]: Good idea to rename this!

File: run.py [hunk 2]
```diff
@@ -13,15 +13,21 @@
...
     cmd = """
         cd {}geckomat
-        model = load('{}');
+        if endsWith('{}','.xml')
+            model = importModel('{}');
+        elseif  endsWith('{}','.mat')
+            model = load('{}');
+        else
+            disp('Model file format is not compatible with GECKO')
+        end
```
Comment (mihai-sysbio) [inline run.py:22]: Sweet!
How about using `error` instead of `disp`?

**Review by mihai-sysbio — APPROVED** [review]

File: run.py (hunk 2 above)
Comment (IVANDOMENZAIN) [reply run.py]: Done!

**Pull request merged**